### PR TITLE
adds updates to 3_get_api

### PR DESCRIPTION
**Pull request by ericnabofa**

### CODE REVIEW

**Comment by Harrison365** [comment]
Hi Eric :) 
It looks like we may have overthought this a bit. All you need to do is require in the endpoints.json file to the controller (or use fs.readFile) and send it back to your tests. You can compare it directly to the endpoints.json by requiring into this file too.

File: app/app.controller.js
Comment (mezzDavies) [inline app/app.controller.js]: If you are going to read the file in then I would use fs promises. That said you could just require the endpoints.json file in at the top of this file and send it as a response.

File: app/app.controller.js
Comment (mezzDavies) [inline app/app.controller.js]: This path is not dynamic so will not work if this repo is used on another machine (eg when hosted) you could use __dirname or follow my hint above to just require the file in instead of using any fs functionality 👍

File: __tests__/app.test.js
Comment (mezzDavies) [inline __tests__/app.test.js]: I would require this in at the top of the file rather than in the test - this is where require statements are by convention

**Review by mezzDavies — APPROVED** [review]
You are close with this but it could do with a tidy up, ideally. At the very least you need to make the path to the endpoints file dynamic in the controller. See my comments below 🙂

File: app/app.controller.js
Comment (ericnabofa) [reply app/app.controller.js]: yes...thanks mezz

File: app/app.controller.js
Comment (ericnabofa) [reply app/app.controller.js]: done

File: app/app.js
Comment (2202Hannah) [inline app/app.js]: We don't technically need this middleware function yet as we only have a get request. Please remove before you merge to main.

**Review by 2202Hannah — APPROVED** [review]
Hi Eric, great job implementing the feedback from Mezz and Harrison. I have made one small in-line comment asking you to remove the middleware function that isn't needed yet in your code. You can merge this in to main when you are ready.

File: app/app.js
Comment (ericnabofa) [reply app/app.js]: alright hannah. thanks
will do so now
sending forth another ticket too 😃 for a pull request review

**Pull request merged**
